### PR TITLE
New package: Bluestacks.AppDataMigrationUtility version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111881&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
>  Manifest Error: Invalid field value. [InstallerType]

**Comment by upintheairsheep** [comment]
Keep open; on vacation.

**Comment by Trenly** [comment]
[Policy] Needs Author Feedback

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/issues/111634

File: manifests/b/Bluestacks/AppDataMigrationUtility/1.0/Bluestacks.AppDataMigrationUtility.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Bluestacks.AppDataMigrationUtility
+PackageVersion: '1.0'
+PackageLocale: en-US
+Publisher: Bluestacks
+PublisherSupportUrl: https://www.bluestacks.com/features/native-gamepad-support.html
+PrivacyUrl: https://www.bluestacks.com/terms-and-privacy.html
+PackageName: AppData Migration Tool
+PackageUrl: http://localhost
```
Comment (stephengillie) [inline manifests/b/Bluestacks/AppDataMigrationUtility/1.0/Bluestacks.AppDataMigrationUtility.locale.en-US.yaml:11]: Please either update or remove the PackageUrl.

Comment (CoolPlayLin) [inline manifests/b/Bluestacks/AppDataMigrationUtility/1.0/Bluestacks.AppDataMigrationUtility.locale.en-US.yaml:11]:
```suggestion
PackageUrl: https://www.bluestacks.com/
```

Comment (CoolPlayLin) [reply manifests/b/Bluestacks/AppDataMigrationUtility/1.0/Bluestacks.AppDataMigrationUtility.locale.en-US.yaml:11]: Why the value of `PackageUrl` is localhost? I don't really understand it.

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by upintheairsheep** [comment]
Appears forbidden for WinGet.

**Pull request closed without merging**